src/unstract/sdk/llm.py

```diff
@@ -120,10 +127,75 @@ def _get_max_tokens(self, reserved_for_output: int = 0) -> int:
         """
         return self.MAX_TOKENS - reserved_for_output
 
-    @deprecated("Use the new class LLM")
+    def set_max_tokens(self, max_tokens: int) -> None:
+        """Set the maximum number of tokens that can be used for the LLM.
+
+        Args:
+            max_tokens (int): The number of tokens to be used at the maximum
+
+            Returns:
+                None
+        """
+        self._llm_instance.max_tokens = max_tokens
+
+    def get_class_name(self) -> str:
+        """Gets the class name of the Llama Index LLM.
+
+        Args:
+            NA
+
+            Returns:
+                Class name
+        """
+        return self._llm_instance.class_name()
+
+    @deprecated("Deprecated class and method. Use LLM instead of ToolLLM")
```

Comment:
@gaya3-zipstack `deprecated` library will throw warning that it is deprecated, we do not need to mention again.

In that case let's change message simply to `Use LLM instead of ToolLLM.`.

Reply:
Done